app/assets/stylesheets/main.css.scss

```diff
@@ -0,0 +1,44 @@
+.navbar {
+  background-color: #4DB4D6; 
+  width: 100%; 
+  overflow: hidden; 
+
+  ul {
+    list-style: none; 
+    
+    li {
+      display: inline; 
+      padding: 0.8em; 
+
+      a { 
+        text-decoration: none; 
+        color: white; 
+      }
+    }
+  }
+}
+
+.links {
+  float: left; 
+ }
+
+.auth {
+  float: right;
+} 
+
+#wrapper {
+  background-color: #d9d9d9; 
+  width: 90%; 
+  margin: 0 auto;  
+}
+
+.details a {
+  //padding: 0.3em;
```

Comment:
the `.details a` selector effectively contains nothing since it's only rule `padding: 0.3em` is currently commented out. Since it's empty, I would just remove that entire selector and the commented-out rule it contains.